app/controllers/publishers_controller.rb

```diff
@@ -161,9 +161,6 @@ def home
     @publisher = current_publisher
 
     uphold_connection = current_publisher.uphold_connection
-    if uphold_connection.blank?
```

Comment:
Let's make this a separate change _just in case_ and watch it for a day

Reply:
@tsmartt I don't have any issue just having this PR go out as it is.  It's all backwards compatible.

Reply:
But to be fair I hadn't even assigned a reviewer, I was just looking at the diff ;)